demos/connector/neo4j/directed-graphsage-on-cora-neo4j-example.ipynb

```diff
@@ -25,7 +25,7 @@
    "source": [
```

Comment:
<p>This demo would be a little more convincing if the labels were read from Neo4j too. I guess that's potentially a little awkward?</p><p> </p><p>(Also applies to undirected.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>